src/components/Gauge/Gauge.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import { bem } from '../../utils';
+import { ContentPlaceholder } from '../ContentPlaceholder';
+import { Context, GAUGE_RADIUS } from '../../constants';
+import styles from './Gauge.scss';
+
+const { block, elem } = bem('Gauge', styles);
+
+interface Props {
+    /** The gauge context (e.g. brand, primary, bad, good etc. - defaults to brand) */
+    context?: Context;
+    /** Defines if progress bar is in loading state */
+    isProgressLoading?: boolean;
+    /** Defines if content part is in loading state */
+    isContentLoading?: boolean;
+    /** Percentage of the progress bar to be filled */
+    percentage?: number;
+    /** Content component */
```

Comment:
Can we make the description a bit more descriptive? For this and all following props.